### ISSUE TITLE
Inertial Touch Scrolling Not Disabled on Touchpad

### ISSUE THREAD

**Issue by tunjan** [flxzt/rnote#1291]
Inertial touch scrolling remains enabled when using a touchpad, despite being explicitly disabled in the settings. The issue does not occur when using a mouse; in that case, inertial scrolling is correctly disabled.

Steps to Reproduce:

    Update to version 0.11.0.
    Disable inertial touch scrolling in the application settings.
    Test scrolling behaviour using a mouse (no inertial scrolling).
    Test scrolling behaviour using a touchpad (inertial scrolling persists).

Expected Behaviour:
Inertial scrolling should be disabled for both the mouse and touchpad when the setting is deactivated.

Actual Behaviour:
Inertial scrolling remains enabled on the touchpad.

**Comment by Kneemund** [comment]
This is technically working as intended, but the wording of the option could be improved to reflect that it only applies to things like touchscreens and drawing tablets.

Internally, this option toggles the [kinetic scrolling](https://docs.gtk.org/gtk4/class.ScrolledWindow.html#touch-support) property of GTK4's `ScrolledWindow`s. This only affects touch input (i.e. not emulated pointers like touchpads).

At a first glance, I can't find a way to disable kinetic scrolling for touchpads. Perhaps GTK4 doesn't offer a way to do that - [at least GTK3 didn't](https://gitlab.gnome.org/GNOME/gtk/-/issues/2263).

**Comment by mahdi4260-1** [comment]
By the way, what is "Inertial Touch Scrolling"? Is it the thing that keeps scrolling even after I remove my finger from the touchpad?

**Comment by Kneemund** [comment]
Yes, inertial/kinetic scrolling adds inertia to the canvas movement when scrolling, i.e. the canvas keeps moving until it fully decelerated.
